### ISSUE TITLE
Typing definition is incompleted, cannot type SelectionEvent function.

### ISSUE THREAD

**Issue by maxime4000** [Simonwep/selection#95]
#### What is the problem?
Updating from 1.7.1 to 2.0.0, I had to fix my typescript code with the new SelectionArea instead of Selection, but before I was using Selection.SelectionEvent to type my onStart/onMove/onStop function, but right now, only SelectionArea is available as typing... Not SelectionArea.SelectionEvent.
I try to import :
`import { SelectionEvent } from "@simonwep/selection-js/src/types"; `
but I get :
`Cannot find module '@utils' or its corresponding type declarations.  TS2307`

#### What is the expected behavior?
Something like this would work :
`import SelectionArea, { SelectionEvent, ChangedElements, SelectionStore, ... } from "@simonwep/selection-js";`

#### Your environment:
```
Version (SelectionArea.version): 2.0.0
Browser:  Typescript compiler ?
OS:  Windows
```

**Comment by Simonwep** [comment]
This is expected, v2 is a major release with breaking changes. See [semver](https://semver.org/).
The type-definition cannot be incomplete since the whole library is now written in TS and type-definitions are automatically generated based on the source-code.

**Comment by maxime4000** [comment]
But how would you import the type SelectionEvent ?

**Comment by maxime4000** [comment]
The problems I suppose is index.ts is only exporting SelectionArea and nothing else.

**Comment by maxime4000** [comment]
I think you misunderstood what I mean @Simonwep. It's not because your code is in typescript that all the typing definition can be imported. If your index.ts is not exporting every types, the definiton of those type are not usable. Right now, only SelectionArea is exported. 
### SelectionArea is exported because of this line :
- File : _src/index.ts:9_ : **export**` default class SelectionArea extends EventTarget<SelectionEvents> {`
- Typing File : _selection.min.d.ts:199_ : `export { SelectionArea as default };`

### Sure there is a work around here which I just try out now :
```
const selection = new SelectionArea({...});
selection.on("start", ({store, event}) => {...}); // this line work because typescript imply SelectionEvent
```
### But if you use it like below, you have to import typing which isn't accessible anymore on v2.
```
const selection = new SelectionArea({...});
const onStart = ({store, event}) => {...}; // store and event are type any
selection.on("start", onStart);
```

**Comment by Simonwep** [comment]
Allright, gotchya - yes you're right. Sorry, they're not exported anymore - that's a problem.

**Comment by Simonwep** [comment]
Fixed and available in [v2.0.1](https://github.com/Simonwep/selection/releases/tag/2.0.1) :)
